utils/appointments.ts

```diff
@@ -0,0 +1,32 @@
+import { AppointmentWithPatient } from '../types/Appointment';
+import dateUtils from './date';
+
+const extractFirstAppointments = (appointments: AppointmentWithPatient[]): AppointmentWithPatient[] => {
+  const startCycle = new Date('2024-01-01T00:00:00Z');
```

Comment:
:green_circle: Si tu suis la convention qui met en majuscule les constantes, tu peux le faire ici aussi : 
```suggestion
  const START_CYCLE_DATE = new Date('2024-01-01T00:00:00Z');
```

Reply:
modification faite